### ISSUE TITLE
Lots of broken images in the specs lately

### ISSUE THREAD

**Issue by impressivewebs** [w3c/csswg-drafts#414]
For example:

https://drafts.csswg.org/css-backgrounds-3/#border-radius

Notice the "Diagram of the inscribed ellipse" image, not loading. It points to:

https://drafts.csswg.org/css-backgrounds-3/corner.png

Which is 404ing (along with what seems to be most other images on that page).

I've noticed this on a number of different CSS spec pages over the past week.

**Comment by SebastianZ** [comment]
Note that the correct path to the image is https://drafts.csswg.org/css-backgrounds-3/images/corner.png.

Sebastian

**Comment by dauwhe** [comment]
Overview.src.html has the correct image paths, but Overview.html doesn't. Introducing myself to the wonderful world of the preprocessor to see what's going on.

**Comment by SebastianZ** [comment]
> Overview.src.html has the correct image paths

FWIW, that was fixed in https://github.com/w3c/csswg-drafts/commit/3109281232008b4a3c0945635dff288806665a8d.

> but Overview.html doesn't. Introducing myself to the wonderful world of the preprocessor to see what's going on.

When Backgrounds 3 is converted to Bikeshed (see #251), then Overview.html will be autogenerated.
Until then, it would be great if you were able to generate an updated version manually.

Sebastian

**Comment by dauwhe** [comment]
@bert-github, any idea why this won't process? If I try to do it through the web interface, I get the message "Error: Incorrect query: missing or illegal field "method". I did fix a couple of well-formedness errors in the HTML source, but that didn't seem to affect anything.

**Comment by bert-github** [comment]
On Thursday 18 August 2016 6:55:43 A.M. CEST Dave Cramer wrote:
> @bert-github, any idea why this won't process? If I try to do it
> through the web interface, I get the message "Error: Incorrect query:
> missing or illegal field "method". I did fix a couple of
> well-formedness errors in the HTML source, but that didn't seem to
> affect anything.

One of our servers had a hardware failure recently and had to be 
rebuilt. We thought we tested all the services running on it, but 
apparently we forgot the postprocessor, because there were two files 
missing from it.

I recreated them and the postprocessor seems to be working now.

Now I just need to get the latest sources and regenerate the editors' 
draft (unless somebody beats me to it...)



Bert
-- 
  Bert Bos                                ( W 3 C ) http://www.w3.org/
  http://www.w3.org/people/bos                               W3C/ERCIM
  bert@w3.org                             2004 Rt des Lucioles / BP 93
  +33 (0)4 92 38 76 92            06902 Sophia Antipolis Cedex, France
